homeassistant/components/zwave_js/api.py

```diff
@@ -1390,7 +1390,15 @@ def forward_finished(event: dict) -> None:
     ]
     connection.subscriptions[msg["id"]] = async_cleanup
 
-    connection.send_result(msg[ID])
+    result = (
```

Comment:
The WS api and service changes don't look related. They should be two separate PRs in my opinion.

- The service PR is about adding the options parameter.
- The WS api PR is about adding progress update it seems.

Reply:
sure, I added them here because they were both added as part of the dependency bump, but you are right that functionally they have different purposes